backend/loaders/ppt_loader.py

```diff
@@ -0,0 +1,25 @@
+from pptx import Presentation
+import os
+
+def load_presentation(presentation_file: bytes, filename: str) -> str:

```

Comment:
please check the other loaders functions, we added meta data, it's in place there